File: Serene_Movies_Task/Serene_Movies_Task.Web/Modules/Default/Genre/GenreColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Serene_Movies_Task.Default.Columns;

[ColumnsScript("Default.Genre")]
[BasedOnRow(typeof(GenreRow), CheckNames = true)]
public class GenreColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int GenreId { get; set; }
    [EditLink]
    public string Name { get; set; }
}

File: Serene_Movies_Task/Serene_Movies_Task.Web/Modules/ServerTypes/Default/GenreColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { GenreRow } from "./GenreRow";

export interface GenreColumns {
    GenreId: Column<GenreRow>;
    Name: Column<GenreRow>;
}

export class GenreColumns extends ColumnsBase<GenreRow> {
    static readonly columnsKey = 'Default.Genre';
    static readonly Fields = fieldsProxy<GenreColumns>();
}